core/src/test/resources/log4j2-test.xml

```diff
@@ -2,14 +2,14 @@
 <Configuration packages="stryker4s" status="WARN">
     <Appenders>
         <TestAppender name="TestAppender"/>
-        <Console name="Console" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{HH:mm:ss.SSS} %-5level %logger{0}: %msg%n"/>
-        </Console>
+        <!--<Console name="Console" target="SYSTEM_OUT">-->
+            <!--<PatternLayout pattern="%d{HH:mm:ss.SSS} %-5level %logger{0}: %msg%n"/>-->
+        <!--</Console>-->
```

Comment:
Why is the console appender removed? Seeing log output during tests can still be useful

Reply:
The issue I have here: it's useful to show log output when just running tests, but when running stryker4s on itself it results in a lot of (unnecessary?) logging because it logs for every mutation. :/

That's why I disabled it, but I guess I shouldn't have committed it.

Reply:
I would prefer that we would have logging during tests. when running stryker on stryker we could of course overwrite the loglevel ;)

Reply:
It is difficult to understand what exactly the tool is doing / which tests are actually being executed, and if there is no way to see the regular test report, then how can I gain any confidence in the mutation reports? #DevEx